Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -487,6 +492,116 @@ customerIssueCarousalView  push =
     ]
 
 
+customerIssuePopupView :: forall w. (Action -> Effect Unit) ->  CustomerIssue -> PrestoDOM (Effect Unit) w
+customerIssuePopupView  push config = 
+      let cardsArray = config.customerIssueCards
+          title = case (head cardsArray) of
+                         Just card -> card.title
+                         Nothing -> ""
+          selectedYes = case (head cardsArray) of
+                       Just card -> (  card.selectedYes )
+                       Nothing -> Nothing
+          yestext = case (head cardsArray) of 
+                        Just card -> card.yesText
+                        Nothing -> ""
+          notext  = case (head cardsArray) of 
+                        Just card -> card.noText
+                        Nothing -> ""
+          subtitle = case (head cardsArray) of
+                        Just card -> card.subTitle
+                        Nothing -> ""
+   in(linearLayout [
+      width MATCH_PARENT
+    , height WRAP_CONTENT
+    , padding $ Padding 16 24 16 24
+    , cornerRadius 6.0
+    , backgroundColor Color.white900
+    , orientation VERTICAL
+    , stroke $ "1,"<>Color.grey800
+    ][
+      textView $ [
+        width MATCH_PARENT
+      , height WRAP_CONTENT
+      , text title
+      , color Color.black800 
+      , gravity CENTER
+      , padding $ PaddingBottom 4
+      ] <> FontStyle.h3 TypoGraphy
+    , textView $ [
+        width MATCH_PARENT
+      , height WRAP_CONTENT
+      , text subtitle 
+      , color  Color.black700
+      , gravity CENTER
+      ] <> FontStyle.paragraphText TypoGraphy 
+    , linearLayout [
+        width MATCH_PARENT
+      , height WRAP_CONTENT
+      , orientation HORIZONTAL
+      , margin $ MarginTop 16
+      ][
+        linearLayout[
+          height WRAP_CONTENT
+        , weight 1.0
+        , gravity CENTER
+        , padding $ Padding 1 1 1 1
+        , cornerRadius 6.0
+        , background $ if selectedYes == Just true then Color.blue900 else Color.grey800
```

Comment:
Create a config on the basis of `selectedYes`, which will include background, accessibility hint, etc

Reply:
not needed, only in one case this view is getting called. Also layout properties aren't added anywhere in this config